src/test/java/hudson/plugins/logparser/LogParserTestCase.java

```diff
@@ -50,15 +48,18 @@ protected void assertLogParserCorrect(Run<?, ?> build, Job<?, ?> project, Result
         assertThat(build.getAction(LogParserAction.class)).as(LogParserAction.class.getSimpleName() + " not found")
                 .isNotNull();
                 
-        boolean aborted = Result.ABORTED.equals(build.getResult());
+        boolean aborted = Result.ABORTED == build.getResult();
         
         LogParserResult parserResult = build.getAction(LogParserAction.class).getResult();
         assertThat(parserResult).isNotNull();
         
         if (!aborted) {
-            assertThat(parserResult.getTotalErrors()).isEqualTo(1);
-            assertThat(parserResult.getTotalWarnings()).isEqualTo(1);
-            assertThat(parserResult.getTotalInfos()).isEqualTo(1);
+            assertThat(new int[] {
```

Comment:
Improved the checking so that in case of an error you can see all three numbers at once.